src/Bridge/Symfony/Bundle/Resources/config/metadata.xml

```diff
@@ -74,6 +89,27 @@
             <argument type="service" id="api_platform.metadata.property.metadata_factory.validator.inner" />
         </service>
 
+        <service id="api_platform.metadata.property.metadata_factory.cached" class="ApiPlatform\Core\Metadata\Property\Factory\CachedPropertyMetadataFactory" decorates="api_platform.metadata.property.metadata_factory" decoration-priority="-1" public="false">
```

Comment:
I've added the apcu service, and the cache is now configurable.

The cache can't be disabled though (except when `symfony/cache` is not available, or when the service specified is invalid). Perhaps we should even warn about extreme slowness when no cache is used.